android/assets/jsons/translations/Traditional_Chinese.properties

```diff
@@ -224,35 +224,23 @@ Provides [amountOfCulture] culture at 30 Influence = 當影響力不低於30時
 Provides 3 food in capital and 1 food in other cities at 30 Influence = 當影響力不低於30時為首都提供3食物，為其他城市提供1食物
 Provides 3 happiness at 30 Influence = 當影響力不低於30時提供3快樂
 Provides land units every 20 turns at 30 Influence = 當影響力不低於30時每20回合提供陸軍單位
- # Requires translation!
-Give a Gift = 
+Give a Gift = 贈禮
 Gift [giftAmount] gold (+[influenceAmount] influence) = 贈予[giftAmount]金錢(+[influenceAmount]影響力)
 Relationship changes in another [turnsToRelationshipChange] turns = 將在[turnsToRelationshipChange]回合後失去對該城邦的影響
- # Requires translation!
-Protected by = 
- # Requires translation!
-Revoke Protection = 
- # Requires translation!
-Revoke protection for [cityStateName]? = 
- # Requires translation!
-Pledge to protect = 
- # Requires translation!
-Declare Protection of [cityStateName]? = 
- # Requires translation!
-Build [improvementName] on [resourceName] (200 Gold) = 
- # Requires translation!
-Gift Improvement = 
+Protected by = 被保護
+Revoke Protection = 撤銷保護
+Revoke protection for [cityStateName]? = 撤銷對[cityStateName]的保護嗎?
+Pledge to protect = 請求保護
+Declare Protection of [cityStateName]? = 宣告對[cityStateName]的保護嗎?
+Build [improvementName] on [resourceName] (200 Gold) = 以[resourceName] (200金) 建造[improvementName] 
+Gift Improvement = 禮物升格
```

Comment:
‘improvement' here is a noun. 'Gift Improvement’ means gifting improvement (like mines) to CS instead of 'giving something better'. So it should be ‘贈送設施'

Reply:
Thanks for the suggestion. Just updated it!